cdtb/__main__.py

```diff
@@ -327,6 +327,8 @@ def command_bin_dump(parser, args):
     if args.json:
         json_dump(binfile.to_serializable(), sys.stdout)
     else:
+        if binfile.linked_files is not None:
+            print(f"linked: {binfile.linked_files}")
```

Comment:
Can we find a name a bit more explicit than just "linked"? I don't remember if it links to actual files. If those are used for dependencies, we could use "dependencies".

Reply:
I don't know how exactly these links are used, but `dependencies` probably makes sense. I was mainly following the existing code's naming and what [ritobin](https://github.com/moonshadow565/ritobin/blob/418af7abf5628afb2cfd621f42f70116ef078b0c/README.md?plain=1#L42) uses.

Reply:
I don't have a strong opinion on this, and you probably know more about those than me. I was just tossing some idea, but we can go with whichever name you choose. ;)